runtime/src/snapshot_utils.rs

```diff
@@ -2255,7 +2277,8 @@ mod tests {
         let max_slot = 123;
         common_create_bank_snapshot_files(temp_snapshots_dir.path(), min_slot, max_slot);
 
-        let highest_bank_snapshot_info = get_highest_bank_snapshot_info(temp_snapshots_dir.path());
+        let highest_bank_snapshot_info =
+            get_highest_bank_snapshot_info(temp_snapshots_dir.path(), true);
```

Comment:
`creating` is `true` here, but I would guess the `highest_bank_snapshot_info.path` would not be an actual file, right?